docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -0,0 +1,912 @@
+---
+lang: en
+title: 'Authentication Tutorial'
+keywords: LoopBack 4.0, LoopBack 4, Authentication, Tutorial
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/authentication-tutorial.html
+summary: A LoopBack 4 application that uses authentication
+---
+
+## Overview
+
+LoopBack 4 has an authentication package `@loopback/authentication` which allows
+you to secure your API endpoints with custom authentication strategies that
+implement a common interface and are registered via an extensionPoint/extension
+pattern.
+
+## Try it out
+
+If you'd like to see the final results of this tutorial as an example
+application, follow these steps:
+
+1.  Clone the `strongloop/loopback4-example-shopping` repository:
+
+    ```sh
+    git clone git@github.com:strongloop/loopback4-example-shopping.git
+    ```
+
+2.  Switch to the directory.
+
+    ```sh
+    cd loopback4-example-shopping
+    ```
+
+3.  Install all the dependencies.
+
+    ```sh
+    npm install
+    ```
+
+4.  Start the MongoDB and the Redis containers:
+
+    ```sh
+    npm run docker:start
+    ```
+
+5.  Build and start the application
+
+    ```sh
+    npm start
```

Comment:
Can we have all steps in one block, such as:
```sh
git clone git@github.com:strongloop/loopback4-example-shopping.git
cd loopback4-example-shopping
npm i
npm run docker:start
npm start
```

Reply:
FYI - I don't have a strong preference whether those commands should be inlined or in the example-shopping repo.

Reply:
@dhmlau , I think we should have it in both.